docs/cloud/netdata-assistant.md

```diff
@@ -0,0 +1,20 @@
+# Alert troubleshooting with Netdata Assistant
+
+The Netdata Assistant is a feature that uses large language models and the Netdata community's collective knowledge to guide you during troubleshooting. It is designed to make understanding and root causing alerts simpler and faster.
+
+# Using Netdata Assistant
```

Comment:
```suggestion
## Using Netdata Assistant
```

i think learn might want this as a h2, at least think thats how the other docs look